app/gitcoinbot/gitcoinbotActions.py

```diff
@@ -0,0 +1,193 @@
+    #     "<br>" +\
+    #     "Learn more at: [https://gitcoin.co](https://gitcoin.co)\n" +\
+    #     ":zap::heart:, {}\n".format("@" + settings.GITHUB_API_USER)
+
+    help_text_response = "I am @{}, a bot that facilitates gitcoin bounties.\n".format(settings.GITHUB_API_USER) + \
+    "\n" +\
+    "<hr>" +\
+    "Here are the commands I understand:\n" +\
+    "\n" +\
+    " * `bounty <amount> ETH` -- receive link to gitcoin.co form to create bounty.\n" +\
+    " * `tip <user> <amount> ETH` -- receive link to complete tippping another github user *<amount>* ETH.\n" +\
+    " * `help` -- displays a help menu\n" +\
+    "\n" +\
+    "<br>" +\
+    "Learn more at: [https://gitcoin.co](https://gitcoin.co)\n" +\
+    ":zap::heart:, {}\n".format("@" + settings.GITHUB_API_USER)
+    return help_text_response
+
+def new_bounty_text(owner, repo, issue_id, comment_text):
+    issue_link = "https://github.com/{}/{}/issues/{}".format(owner,
+                                                            repo,
+                                                            issue_id )
+    bounty_amount = parse_comment_amount(comment_text)
+    # prod
+    # bounty_link = "https://gitcoin.co/funding/new?source={}&amount={}".format(
+    #     issue_link, bounty_amount
+    # )
+    #dev
+    bounty_link = "http://localhost:8000/funding/new?source={}&amount={}".format(
+        issue_link, bounty_amount
+    )
+
+    new_bounty_response = "To create the bounty please [visit this link]({}).\n".format(bounty_link) +\
+    "\n" +\
+    "PS Make sure you're logged in to Metamask!"
+    return new_bounty_response
+
+
+def parse_comment_amount(comment_text):
+    re_amount = r'\d+\.?\d+'
+    result = re.findall(re_amount, comment_text)
+
+    return result[0]
+
+def parse_tippee_username(comment_text):
+    re_username = r'[@][a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}'
+    username = re.findall(re_username, comment_text)
+
+    return username[-1]
+
+def tip_text(comment_text):
+    tip_amount = parse_comment_amount(comment_text)
+    username = parse_tippee_username(comment_text)
+
+    # prod
+    # tip_link = 'https://gitcoin.co/tip/?amount={}&username={}'.format(
+    #   tip_amount, username)
+    # dev
+    tip_link = 'http://localhost:8000/tip/?amount={}&username={}'.format(
+        tip_amount, username)
+    tip_response = 'To complete the tip, please [visit this link]({}).'.format(
+        tip_link)
+
+    return tip_response
+
+def claim_bounty_text(comment_text, sender):
+    #prod
+    # claim_link = 'https://gitcoin.co/funding/claim?source={}&githubUsername={}'.format(
+    #     url, sender
+    # )
+    #dev
+    claim_link = 'http://localhost:8000/claim?source={}&githubUsername={}'.format(
+        url, sender
+    )
+    claim_response = 'To finish claiming this bounty please [visit this link]({})'.format(
+        claim_link
+    )
+
+def confused_text():
+    response = 'Sorry I did not understand that request. Please try again'
+    return response
```

Comment:
I'd just return this string versus assigning it to a var and returning that.  You might not even need this, since it's occurring in one place.